### PR TITLE
Link Placeholders

### PR DESCRIPTION
**Pull request by annstriganova**

### CODE REVIEW

File: rule-engine/links.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+To display the above links in inline mode (where label is displayed instead of URL), make sure the following settings are specified in the Web Notification editor:
+
+* Slack
+ 
+   **Parse Mode** must be set to *Default* or `false`.
+   
+* HipChat Data Center
+ 
+   **Message Format** must be set to *html*.
+
```
Comment (rodionos) [inline rule-engine/links.md]: Add a table with 3 columns with examples: Placeholder, Raw Link, Inline Link. See 4884. Exclude placeholders where Raw and Inline links are the same. We only need to show how raw links change when inlined.

Comment (rodionos) [reply rule-engine/links.md]: Also add a screenshot in slack that show how entity link is presented when its inlined and when its raw.

File: rule-engine/links.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+ 
+   **Message Format** must be set to *html*.
+   
+|Placeholder|Raw Link|Inline Link|
+|---|---|---|
+|ruleLink|`<a href="https://atsd_host:8443/rule/edit.xhtml?name=nmon_cpu"> nmon_cpu </a>`|![](images/links-hipchat-ruleLink.png)|
+|chartLink|`<a href="https://atsd_host:8443/portals/series?metric=docker&entity=nurswgvml007...."> Default </a>`|![](images/links-hipchat-chartLink.png)|
+|csvExportLink|`<a href="https://atsd_host:8443/export?settings=%7B%22m%22%3A%22docker%22%2C%22e%22%3A%22nurswgvml007...."> CSV Export </a>`|![](images/links-hipchat-csvExportLink.png)|
+|htmlExportLink|`<a href="https://atsd_host:8443/export?settings=%7B%22m%22%3A%22docker%22%2C%22e%22%3A%22nurswgvml007...."> HTML Export </a>`|![](images/links-hipchat-htmlExportLink.png)|
+|entityLink|`<a href="https://atsd_host:8443/entities/nurswgvml007"> nurswgvml007 </a>`|![](images/links-hipchat-entityLink.png)|
```
Comment (annstriganova) [inline rule-engine/links.md]: Tried to use images showing inline links, it seems that it's bad idea. Replace screenshots with 

nmon_cpu
Default
CSV Export
HTML Export
nurswgvml007

?

Comment (rodionos) [reply rule-engine/links.md]: ok, maybe no screenshots

**Pull request merged**